spec/lib/sidekiq/throttled/strategy_spec.rb

```diff
@@ -199,6 +206,208 @@
     end
   end
 
+  describe "#requeue_throttled" do
+    let(:sidekiq_config) do
+      if Gem::Version.new(Sidekiq::VERSION) < Gem::Version.new("7.0.0")
+        Sidekiq::DEFAULTS
+      else
+        Sidekiq::Config.new(queues: %w[default]).default_capsule
+      end
```

Comment:
We dropped sidekiq < 7 support - so we can skip it here too :

```suggestion
      Sidekiq::Config.new(queues: %w[default]).default_capsule
```